frontend/src/InteractiveSearch/InteractiveSearch.js

```diff
@@ -65,6 +65,7 @@ const columns = [
   },
   {
     name: 'customFormatScore',
+    columnLabel: () => translate('CustomFormatScore'),
```

Comment:
Do we need `columnLabel` here? Icon already has a title.

Reply:
You already changed it, but those are required because the columns are optional, in this case it's not.